SDPi_Supplement/asciidoc/volume2/gateways/tf2-ch-b-gateway-msh-mapping.adoc

```diff
@@ -0,0 +1,50 @@
+===== MSH - Message Header Segment
+The HL7 Message Header (MSH) segment requires a mapping between the SDC MDIB content and the MSH segment fields.
+
+If not differently specified in this section, the MSH segment fields shall be in compliance with the [PCD-01] or [PCD-04] transaction, retrospectively, as described in the <<ihe_pcd_tf-2_2019>>.
+
+[NOTE]
+====
+If the SDC MDIB contains several MDS elements that could operate in different *pm:MdsState/@OperatingMode* states, there shall be a separate [PCD-01] message per MDS.

```

Comment:
See https://github.com/IHE/sdpi-fhir/pull/163/files#r1012884811

Reply:
Plus, I think we can replace every occurrence of _SDC MDIB_ with <<acronym_mdib>>, which should clarify that whenever mentioned in this supplement, it refers to SDC's concept of an MDIB.

Reply:
Done - replaced "SDC MDIB" with <<acronym_mdib>>.